src/coreComponents/integrationTests/linearAlgebraTests/CMakeLists.txt

```diff
@@ -25,7 +25,8 @@ foreach(test ${LAI_tests})
 if ( ENABLE_MPI )
   geos_add_test( NAME ${test_name}
                  COMMAND ${test_name} -x ${nranks}
-                 NUM_MPI_TASKS ${nranks} )
+                 NUM_MPI_TASKS ${nranks}
+                 NUM_OMP_THREADS 1 )
```

Comment:
On Dane `testDofManager` and `testKrylovSolvers` both ran over 1500s and were killed by ctest, almost certainly due to threading issues with the algebra libraries and perhaps BLAS. With this change they run in seconds.